openid-4-verifiable-presentations-1_0.md

```diff
@@ -2846,29 +2631,27 @@ GET /authorize?
   &id_token_type=subject_signed
   &client_id=x509_san_dns%3Aclient.example.org
   &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
-  &presentation_definition=...
```

Comment:
@danielfett should this be replaced by a `dcq_queryl=...`?

Reply:
Good point - yes